parts/linux/cloud-init/artifacts/components.json

```diff
@@ -48,8 +48,7 @@
       "multiArchVersionsV2": [
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=containernetworking/azure-cni",
-          "latestVersion": "v1.5.35",
-          "previousLatestVersion": "v1.5.32",
```

Comment:
you need to remove the `previousLatestVersion` from containerImagePrefetch if you decide to remove it here